install_config/aggregate_logging.adoc

```diff
@@ -49,12 +51,20 @@ this topic use `logging` as an example project, but you can pick whichever name
 +
 ====
 ----
-$ oc new-project logging
+$ oadm new-project logging --node-selector label=value
 ----
 ====
++
+[NOTE]
+====
+The selector in the example is optional. It can be used to specify a
+region or other selector you would like to use to restrict where the
+logging infrastructure deploys.
+====
+
```

Comment:
This is a neat feature, but it's much easier to show it up front than it is to explain editing it in after creation. It only exists on the admin version of new-project.